### DISCUSSION TITLE
storage s3 error

### DISCUSSION THREAD

**Discussion by wsdnathan** [orchidsoftware/platform#1870] [Q&A] [answered]
When I try to store to this configured filesystem I get this error:
`Class "League\Flysystem\AwsS3v3\AwsS3Adapter" not found`
Flysystem 1.1 already exists required by Laravel 8 in /vendor/league/flysystem/
What am I missing here? Thanks

**Comment by wsdnathan** [comment, marked as answer]
Resolved issue by installing `composer require league/flysystem-aws-s3-v3 "^1.0"`

**Reply by tabuna** [reply]
This is not an issue with the Orchid package. This is stated in the Laravel documentation https://laravel.com/docs/8.x/filesystem

![image](https://user-images.githubusercontent.com/5102591/133332084-4918d98f-db5e-4015-aa25-d0cee77e64eb.png)
